Jenkinsfile

```
@@ -27,6 +27,12 @@ spec:
     command:
     - cat
     tty: true
+  - name: aws-cli
+    image: 'amazon/aws-cli:2.11.20'
```

The image tag `amazon/aws-cli:2.11.20` is over 2 years old.
```suggestion
    image: 'amazon/aws-cli:2.23.2'
```